### PR TITLE
[Microsite][Org] Adding enums for wallet microsite 

### PR DESCRIPTION
**Pull request by amgadani**
Related PR for org: https://github.com/Uniswap/org/pull/88/

### CODE REVIEW

File: src/primitives.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ export enum EventName {
   NFT_TRENDING_ROW_SELECTED = 'Trending Row Selected',
   PAGE_CLICKED = 'Page Clicked',
   PAGE_VIEWED = 'Page Viewed',
+  PAGE_TRANSITION = 'Page Transition',
```
Comment (adjkant) [inline src/primitives.ts:37]: What's this used for exactly? Would it make more sense to log two separate page viewed events?

Comment (amgadani) [reply src/primitives.ts:37]: originally the goal was to see where people were entering the microsite from, and where they were going to. For now i think we're punting on that so its really just to separate the the event where a user lands on the microsite, and then the event where they exit the microsite (page transition is used for the latter, and includes a session duration for the site). theres maybe a better way to do the current event but i _believe_ the goal eventually will be to have a transition_from_page and a transition_to_page

Comment (adjkant) [reply src/primitives.ts:37]: Got it! It sounds like it's meant to track external linking basically, which makes sense to separate from a page-viewed event. 

You could capture the session duration by logging the page view + link click using the two events above. Would that work?

If not and we need a separate event type, I think the name could be a bit more specific to be `SITE_EXIT` or something.

**Comment by just-toby** [comment]
can we close this PR?

**Comment by adjkant** [comment]
sounds like we can yeah, can reopen / make a new PR if still relevant

**Pull request closed without merging**